tests/test_annotations.py

```diff
@@ -150,8 +153,8 @@ def test_download_over_cache(self):
         assert an.is_cached()
 
 
-@pytest.mark.xfail  # HomSap annotation not currently available
-class TestGetChromosomeAnnotations(tests.CacheReadingTest):
+# @pytest.mark.xfail  # HomSap annotation not currently available
+class TestGetChromosomeAnnotations(unittest.TestCase):
```

Comment:
I think this really should inherit from `tests.CacheReadingTest`. (and unittest.TestCase isn't needed anymore)